### DISCUSSION TITLE
Request to bring back Tuning / pitch in Musescore 4

### DISCUSSION THREAD

**Discussion by Harpsichordfan** [musescore/MuseScore#18252] [Ideas]
In Musescore 3, I used the option Synthesiser to adjust the main pitch in Hz. Unfortunatly, in Musescore 4, this option doesn't exist anymore. It is possible, but then you have to select all notes and change the pitch manually every time you wrote new notes. 
It would be very helpful if a function to control the main pitch in Hz would come back to Musescore 4, so that I, and many others, can enjoy the benefits of the new version.

**Comment by cbjeukendrup** [comment]
I've turned this into an issue / feature request: #18257 
So it's added to our list of requests!
